pkg/secrets/check_rights_nix_test.go

```diff
@@ -20,33 +20,49 @@ func setCorrectRight(path string) {
 }
 
 func TestWrongPath(t *testing.T) {
-	require.NotNil(t, checkRights("does not exists"))
+	require.NotNil(t, checkRights("does not exists", false))
 }
 
 func TestGroupOtherRights(t *testing.T) {
 	tmpfile, err := ioutil.TempFile("", "agent-collector-test")
 	require.Nil(t, err)
 	defer os.Remove(tmpfile.Name())
 
+	allowGroupExec := false
+
 	// file exists
-	require.NotNil(t, checkRights("/does not exists"))
+	require.NotNil(t, checkRights("/does not exists", allowGroupExec))
 
 	require.Nil(t, os.Chmod(tmpfile.Name(), 0700))
-	require.Nil(t, checkRights(tmpfile.Name()))
+	require.Nil(t, checkRights(tmpfile.Name(), allowGroupExec))
 
 	// we should at least be able to execute it
```

Comment:
Can we execute w/o being able to read?

Reply:
no, we need both.

Reply:
So perhaps it's time to update the `checkRights()` function and the tests, then?

Reply:
I see the code tests `0500` now, but there is no fix or test that `0100` alone is not enough. Up to you guys whether you want to fix this: it is not a security concern, just a failure to read _and_ execute.

Reply:
Did the change too fast:
* If the "command" is a binary `0100` is enough
* but if it is a script `bash`, `python`... `0500` is needed. 
So I will put back: 0100 in the test